### PR TITLE
fix: analytics checkout

### PR DESCRIPTION
**Pull request by ahuininga-orisha**
When tweakwise analytics is enabled it gives an error because an string is send instead of an store code.

### CODE REVIEW

File: Observer/TweakwiseCheckout.php [hunk 1]
```diff
@@ -43,9 +43,9 @@ public function __construct(
      * @return void
      * @throws \Magento\Framework\Exception\NoSuchEntityException
      */
-    public function execute(Observer $observer)
+    public function execute(Observer $observer) : void
```
Comment (jansentjeu) [inline Observer/TweakwiseCheckout.php]: Code style; remove space. 

`public function execute(Observer $observer): void`

File: Observer/TweakwiseCheckout.php [hunk 2]
```diff
@@ -60,7 +60,7 @@ public function execute(Observer $observer)
      * @return void
      * @throws \Magento\Framework\Exception\NoSuchEntityException
      */
-    private function sendCheckout($items)
+    private function sendCheckout($items) : void
```
Comment (jansentjeu) [inline Observer/TweakwiseCheckout.php]: Code style; remove space.

`private function sendCheckout($items): void`

**Review by jansentjeu — CHANGES_REQUESTED** [review]
Also add return types to other functions from the [PR](https://github.com/EmicoEcommerce/Magento2Tweakwise/pull/229)

**Comment by ahuininga-orisha** [comment]
@jansentjeu I've added the return types

**Review by jansentjeu — APPROVED** [review]

**Pull request merged**